README.md

```diff
@@ -22,6 +22,27 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
 
 **Notice**: This will produce a debug build with several debugging flags enabled. If you want to disable them, modify `config.js`.
 
+-   Mac building with Windows
+-   Make sure git `git lfs` extension is on your path
+-   Run `git lfs pull` to download sound assets
+-   Make sure `ffmpeg` is on your path
+-   Install Node.js and Yarn
+-   Be sure to run cmd as ADMIN! Otherwise this will not work.
+-   Run yarn in the root folder, then run 'yarn' in the `gulp/` folder
+-   Cd into `gulp` and run `yarn gulp main.standalone`
+-   Go into 'tmp_standalone_files' folder with file explorer and navigate into shapez.io-standalone-darwin-x64 or x32 folder
+-   Navigate into shapez.io-standalone.app/Contents/Frameworks folder
```

Comment:
I think everything before this line can be dropped since its already in the regular readme

Reply:
Maybe, but then i need to exclude run cmd as admin in that section, otherwise it wont work.

Reply:
Well then just write `Execute the regular build steps as admin`